lib/rubocop/rspec/description_extractor.rb

```diff
@@ -68,11 +68,7 @@ def documented_constant
         end
 
         def cop_subclass?
-          # YARD superclass resolution is a bit flaky: All classes loaded before
-          # RuboCop::Cop::WorkaroundCop are shown as having RuboCop::Cop as
-          # superclass, while all the following classes are listed as having
-          # RuboCop::Cop::RSpec::Cop as their superclass.
```

Comment:
Now it's more or less clear. There's a better way - pass `RuboCop::Cop::RSpec::Cop`'s path as the first argument to `YARD.parse` to avoid its confusion.